GeoMesaQuery.md

```diff
@@ -0,0 +1,81 @@
+# GeoMesa Indexing and Querying #
+
+## The GeoMesa Index ##
```

Comment:
one more note: I'm not including timestamps (we use logical time in accumulo) or visibilities (although they will get included if you mark your data appropriately)